### ISSUE TITLE
Veyon crashes

### ISSUE THREAD

**Issue by bf28** [veyon/veyon#127]
Hi:

On 9/26/17 I was able to configure veyon on all the computers we have at our computer lab and it was fine. However, today 9/27/17 veyon master keeps crashing despite restarting it several times. See screenshot below. Should I uninstall and reinstall veyon? 

Thanks
![capture](https://user-images.githubusercontent.com/32274269/30923572-360a2160-a37a-11e7-8f81-09e18380aab9.PNG)

**Comment by bf28** [comment]
Hello:

It looks like the issue fixed itself. Thanks

**Comment by bf28** [comment]
Actually, the issue above still persists. I have to restart veyon master a few times before I can finally monitor the rest of the screens. 

I'm also having issues in sharing my screen with the rest of the computers in the lab. This is what I get: 
![black screen](https://user-images.githubusercontent.com/32274269/30975727-f5add5b8-a440-11e7-835a-39826afba5c9.PNG)

Shouldn't my screen automatically appear on the rest of the screens? Am I doing something wrong? Thank you and look forward to a quick resolution.

**Comment by shadowserge** [comment]
I have the same issue, but I noticed that it usually happens when the students are moving the mouse or using the keyboard so I found a really simple work around: just lock all the computers first and then press the full screen/window demo button without unlocking them. Even if some computers show the black screen, it fixes itself in a matter of seconds.

**Comment by AlessandroPhilips** [comment]
how did you all configurate the veyon itn not working on my clients when i try to connect there is an error that says ....(host computer) try to connect but is not verifided help me with this issue pls

**Comment by bf28** [comment]
Shadowserge, the issue shows up even when there are no classes in session. Today, for example, I had to restart veyon master several times before it finally became stable. I just don't understand why it does that. I tried your suggestion, but after locking the computers and pressing full screen, the screens still remained black. I really want this feature to work, since my classroom doesn't have a smartboard.
